src/components/modal/v2/parts/Calculator.jsx

```diff
@@ -218,7 +218,7 @@ const Calculator = ({ setExpandedState, calculator, aprDisclaimer, useV4Design,
                     />
                 </div>
             ) : null}
-            {country === 'US' && (
+            {country === 'US' && !error && (
```

Comment:
```suggestion
            {country === 'US' && (
```
I just realized that the check you added here removed the disclaimer under the error message and we always want that to show up regardless so we'll want to revert the change.